var/spack/repos/builtin/packages/h5utils/package.py

```diff
@@ -11,10 +11,11 @@ class H5utils(AutotoolsPackage):
     scientific data in the free, portable HDF5 format."""
 
     homepage = "http://ab-initio.mit.edu/wiki/index.php/H5utils"
-    url      = "http://ab-initio.mit.edu/h5utils/h5utils-1.12.1.tar.gz"
-    list_url = "http://ab-initio.mit.edu/h5utils/old/"
+    url      = "https://github.com/NanoComp/h5utils/releases/download/1.13.1/h5utils-1.13.1.tar.gz"
```

Comment:
Love the switch to `https`!

Reply:
This was replaced (and list_url was removed already), it was in an old commit.

Reply:
@tldahlgren your requested changes are applied.